### DISCUSSION TITLE
Adding an email to a repo

### DISCUSSION THREAD

**Discussion by Norma333** [community/community#38667] [Other Feature Feedback, Questions, & Ideas] [answered]
I need help adding an email to a new repo that I just created. My developer needs me to add his email to it so that he can push the code to that specific repo.

What are the steps to do this?

**Comment by SilasRodrigues19** [comment, marked as answer]
@Norma333 `Access the repo` -> `Settings` -> `Collaborators` -> `Add People`

**Reply by SilasRodrigues19** [reply]
![image](https://user-images.githubusercontent.com/47532703/200846465-39d13a9b-5bfe-40c4-8046-e707f2c69d78.png)

**Comment by Norma333** [comment]
Thank you soooo much!
